langkit/examples/v0_1_0_changes.md

```diff
@@ -0,0 +1,87 @@
+# Changes
+
+## Glossary
+
+- Feature Modules: Stuff like `toxicity`,`textstat`, `sentiment`, etc.
+- Feature Presets: A set of feature modules. Possible presets: `light_features`,`llm_features`,`security`,`leakage`, etc.
+
+## No auto-init
+
+Feature modules are no longer auto-initialized. This means that you need to call init for every module you want to use. This is a breaking change.
+
+Example:
+
+```python
+from langkit import toxicity, extract
+
+schema = toxicity.init()
+
+enhanced_df = extract(df, schema=schema)
+```
+
+## Every module's init returns a schema
+
+Every module's init function now returns a schema. The schema should contain only the UDFs that the respective module provides (no global udf registry). This is a breaking change.
+
+Example:
+
+```python
+from langkit import toxicity, sentiment, extract
+
+sentiment_schema = sentiment.init()
+toxicity_schema = toxicity.init() 
+
+enhanced_df = extract(df, schema=schema)
+```
+
+## Schema Mergeability
+
+Schemas are mergeable, and that should be the way to build Feature Presets (built-in or custom)
+
+```python
+from langkit.presets import llm_features
+llm_schema = llm_features.init(config: LangkitConfig)
+enhanced_df = extract(df, schema=my_schema)
+```
+
+```python
+from langkit import toxicity, sentiment, extract
+
+sentiment_schema = sentiment.init()
+toxicity_schema = toxicity.init() 
+
+my_schema = sentiment_schema.merge(toxicity_schema)
+
+enhanced_df = extract(df, schema=my_schema)
+```
+
+## Config centralization
+
+Every feature module should be fully customizable through a `LangkitConfig` object.
```

Comment:
We need to break up `LangKitConfig`. Each module should have its own config, perhaps subclassed from a parent with a few common parameters (prompt & response column names, prompt & response languages).